.github/workflows/check.yml

```diff
@@ -41,3 +41,5 @@ jobs:
           | jq '.packages | .[] | .name' \
           | rg --invert-match 'atleta-(node|runtime)' \
           | xargs -I {} cargo clippy --package {} --all-targets --all-features -- --deny warnings --deny missing_docs
+    - name: Run shellcheck
+      run: find "$GITHUB_WORKSPACE/deploy" "$GITHUB_WORKSPACE/scripts" -name '*.sh' -print0 | xargs -0 -I {} shellcheck --exclude=SC1090 --exclude=SC1091 {}
```

Comment:
Do we really need these `--exclude`?
If yes, then we should try to configure it through some project-scoped configuration file like `.shellcheckrc`.